assets/image_index_preview.publish.js

```diff
@@ -45,7 +45,7 @@ jQuery(document).ready(function($) {
 				if (link.data('path') != null) {
 					path = link.data('path');
 					filename = link.text();
-					file = path.replace(root, '').replace('/workspace','') + filename;
+					file = path.replace(root,'').replace('/workspace/','') + '/' + filename;
```

Comment:
Why don't you just replace `/workspace/` with `/` instead of an empty string?

```js
path.replace(root, '').replace('/workspace/', '/') + filename;
```

While `root` might not be part of the filename (relative links), all uploaded files reside inside `/workspace` given the way Symphony's upload fields work.